src/app/search/search-workflow-table/search-workflow-table.component.html

```diff
@@ -19,7 +19,10 @@
             }}</a
           >
           <div class="truncate-text-2">
-            <app-ai-bubble *ngIf="workflow?.source.topicSelection === TopicSelectionEnum.AI" [isPublic]="true"></app-ai-bubble>
+            <app-ai-bubble
+              *ngIf="workflow?.source.topicSelection === TopicSelectionEnum.AI && !workflow?.source.approvedAITopic"
```

Comment:
Would be great to encapsulate/centralize this logic, but creating a pipe to do it is so heavy...